addons/hr_timesheet/report/project_report_view.xml

```diff
@@ -23,6 +23,7 @@
             <field name="inherit_id" ref="project.view_task_project_user_pivot"/>
             <field name="arch" type="xml">
                 <xpath expr="//field[@name='project_id']" position='after'>
+                    <field name="overtime" widget="timesheet_uom"/>
```

Comment:
I'd say it would worth separating the diff into multiple commits (services app in one, portal in another, mail in another and rating in a last one)
The title of the commit should also start by a verb.
Thanks 🙂